webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalHooks.js

```diff
@@ -0,0 +1,14 @@
+import { useState, useContext } from 'react';
+import { ModalContext } from './ForemanModal';
+
+export const useModalState = initialModalState => {
```

Comment:
It was confusing for me, I just realized it's only there for the storybook.
Would it make sense putting it inside the storybook file?

Reply:
Ok, moving